### DISCUSSION TITLE
有成功部署的吗

### DISCUSSION THREAD

**Discussion by Jesus-Yu** [xai-org/grok-1#179] [General]
有人成功了吗

**Comment by oziasly** [comment]
俺也想问问

**Comment by az13js** [comment]
这东西个人玩的话没个几百万元买GPU的话跑不动吧

**Reply by leige6** [reply]
issue里有个人硬件满足了，还是没跑起来，在提issue问呢

**Comment by Lz020316** [comment]
+1 
有部署成功的不

**Comment by nijisakai** [comment]
我的已经在跑，但是卡在info
#210

**Reply by streem0025** [reply]
我也是，临时文件太大了，估计磁盘不够

**Comment by ppkdch54** [comment]
看了硬件需求后，默默的删了正在下载的模型

**Comment by JiayiFu** [comment]
We have successfully deployed on the 8*A800(80G) machine....

**Reply by nijisakai** [reply]
一样的硬件，请问有做过什么代码调整么？我一直卡在INFO界面。

**Comment by realgump** [comment]
I have successfully deployed.

**Reply by nijisakai** [reply]
can i see the output of the terminal, and how long it took to give answer.
This is mine, stuck in INFO. (8xA800 80G, 500G RAM and 4T of free disk)
![image](https://github.com/xai-org/grok-1/assets/29118280/cbebc717-f3b9-4a64-b675-61ff54a19123)

**Reply by realgump** [reply]
Emm... It does takes a really long time to give the final anwser. I think maybe about 10 minutes.

**Reply by streem0025** [reply]
It's stuck here because the model is generating temporary files, and the size of these files is very large. I've already generated nearly 400GB of files, and it's not finished yet.

**Reply by nijisakai** [reply]
> It's stuck here because the model is generating temporary files, and the size of these files is very large. I've already generated nearly 400GB of files, and it's not finished yet.

but it stuck here for 8 hours yet.

**Reply by streem0025** [reply]
You should first make sure that your disk capacity is sufficient. It's best to use a large-capacity solid-state drive (SSD). If you use a mechanical hard drive, the speed will be slower.

**Comment by nijisakai** [comment]
> Emm... It does takes a really long time to give the final anwser. I think maybe about 10 minutes.

It stuck here for serveral hours. Is it the right output(many INFOs)?

**Reply by realgump** [reply]
The INFOs seems normal. Have you observe the usage of memory, or other metrics?

**Reply by nijisakai** [reply]
Memory usage is 50% of 500G RAM and I set 500G of /dev/shm
CPU usage is 800% from 'top'

**Comment by xiciliu** [comment]
https://www.yuque.com/u21594513/pwky0v/fe0lcl5m7oqwfy4l?singleDoc# 《grok-1安装指南》

**Reply by xiciliu** [reply]
![图片](https://github.com/xai-org/grok-1/assets/6055072/c84b72f7-d227-4df1-a8ef-2a491b7e673a)
进群交流
